lib/fathead/vuejs/output.txt

```diff
@@ -0,0 +1,132 @@
+Vue extend	R	Vue.extend										
+extend	R	Vue.extend										
+Vue.nextTick	A			Global API						https://vuejs.org/images/logo.png	<section class="prog__container"><pre><code>Vue.nextTick( [callback, context] )</code></pre><p><span class="prog__sub">Arguments:</span></p>\n<ul>\n<li><code>{Function} [callback]</code></li>\n<li><code>{Object} [context]</code></li>\n</ul>\n<p><span class="prog__sub">Usage:</span></p><p>Defer the callback to be executed after the next DOM update cycle. Use it immediately after you&#x2019;ve changed some data to wait for the DOM update.</p><pre><code>// modify data\nvm.msg = 'Hello'\n// DOM not updated yet\nVue.nextTick(function () {\n  // DOM updated\n})\n</code></pre><p><span class="prog__sub">See also:</span> Async Update Queue</p>\n</section>	https://vuejs.org/v2/api/#Vue.nextTick
+Vue nextTick	R	Vue.nextTick										
+nextTick	R	Vue.nextTick										
+Vue.set	A			Global API						https://vuejs.org/images/logo.png	<section class="prog__container"><pre><code>Vue.set( object, key, value )</code></pre><p><span class="prog__sub">Arguments:</span></p>\n<ul>\n<li><code>{Object} object</code></li>\n<li><code>{string} key</code></li>\n<li><code>{any} value</code></li>\n</ul>\n<p><span class="prog__sub">Returns:</span> the set value.</p>\n<p><span class="prog__sub">Usage:</span></p>\n<p>Set a property on an object. If the object is reactive, ensure the property is created as a reactive property and trigger view updates. This is primarily used to get around the limitation that Vue cannot detect property additions.</p>\n<p><span class="prog__sub">Note the object cannot be a Vue instance, or the root data object of a Vue instance.</span></p>\n<p><span class="prog__sub">See also:</span> Reactivity in Depth</p>\n</section>	https://vuejs.org/v2/api/#Vue.set
+Vue set	R	Vue.set										
+set	R	Vue.set										
+Vue.delete	A			Global API						https://vuejs.org/images/logo.png	<section class="prog__container"><pre><code>Vue.delete( object, key )</code></pre><p><span class="prog__sub">Arguments:</span></p>\n<ul>\n<li><code>{Object} object</code></li>\n<li><code>{string} key</code></li>\n</ul>\n<p><span class="prog__sub">Usage:</span></p>\n<p>Delete a property on an object. If the object is reactive, ensure the deletion triggers view updates. This is primarily used to get around the limitation that Vue cannot detect property deletions, but you should rarely need to use it.</p>\n<p><span class="prog__sub">Note the object cannot be a Vue instance, or the root data object of a Vue instance.</span></p>\n<p><span class="prog__sub">See also:</span> Reactivity in Depth</p>\n</section>	https://vuejs.org/v2/api/#Vue.delete
+Vue delete	R	Vue.delete										
```

Comment:
Is there anything else that is removed by the query parser? I'll make these changes to Mongoose fathead too